package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "eslint": "eslint --ignore-path .gitignore .",
     "dev": "export NODE_ENV=development && nodemon src/temp-server.js ",
-    "test": "export NODE_ENV=testing && nodemon src/temp-server.js "
+    "test": "NODE_ENV=testing nyc mocha ./src/test/*.js  --timeout 200000 --exit",
```

Comment:
@nziokaivy we are going to write many tests I recommend to rename your `test` directory to be `tests` and update your `package.json` scripts to be /src/**tests**/*.js

Reply:
@NiyoEric Noted. Working on the requested changes